lib/svg-preview-view.js

```diff
@@ -27,6 +27,17 @@ class SvgPreviewView extends View {
             this.text('transparent')
           })
         })
+        this.div({ class: 'image-controls-zoom-group' }, () => {
+          this.a({ outlet: 'zoomInButton', class: 'image-controls-zoom-in' }, () => {
+            this.text('+')
+          })
+          this.a({ outlet: 'zoomOutButton', class: 'image-controls-zoom-out' }, () => {
+            this.text('-')
+          })
+          this.a({ outlet: 'zoomResetButton', class: 'image-controls-zoom-reset' }, () => {
+            this.text('1:1')
+          })
+        })
```

Comment:
The only thing you need to change to make them look good is to make them `<button>` tags instead of `<a>` ;-)

The styling ist standard atom-ui styling. See: [atom/atom-ui](https://github.com/atom/atom-ui)